com.unity.shadergraph/Editor/Data/Nodes/Math/Vector/TransformNode.cs

```diff
@@ -189,7 +189,7 @@ public void GenerateNodeCode(ShaderStringBuilder sb, GenerationMode generationMo
             {
                 if (conversion.to == CoordinateSpace.World)
                 {
-                    transformString = string.Format("mul(UNITY_MATRIX_I_V, $precision4({0}, 1)).xyz", inputValue);
```

Comment:
Won't this make TransformViewToWorld wrong for position? I believe this needs a conditional like the otehr cases (see ViewToObject below).

Reply:
Good catch.  Fixed.  I think this also means the test images don't need to be updated, but I'm re-running to confirm.

Reply:
Actually, if the images don't need to be updated, this implies we don't have a test for both scenarios. Maybe adding new nodes might be good.